health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SetupMicroplan.js

```
@@ -94,6 +94,22 @@ const SetupMicroplan = ({ hierarchyType, hierarchyData }) => {
     setMicroplanConfig(MicroplanConfig(params, null, isSubmitting, null, hierarchyData));
   }, [totalFormData, isSubmitting]);
 
+  useEffect(() => {
+    const handleCheckingEvent = () => {
+      console.log("Checking event triggered");
+      const newKey = parseInt(new URLSearchParams(window.location.search).get("key")) || 1;
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)